Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -525,4 +541,76 @@ namespace Multiplayer
         constexpr bool autoRequeue = true;
         m_connectionEvent.Enqueue(AZ::SecondsToTimeMs(retrySeconds), autoRequeue);
     }
+
+    void MultiplayerEditorSystemComponent::PopulateEditorGlobalContextMenu(QMenu* menu, const AZ::Vector2& point, [[ maybe_unused ]] int flags)
+    {
+        menu->setToolTipsVisible(true);
+
+        m_contextMenuViewPoint = point;
+
+        QAction* action = nullptr;
+
+        action = menu->addAction(QObject::tr("Create multiplayer entity"));
```

Comment:
NP: Do we want "Create Multiplayer Entity" here? Not sure what the style requirement is.

Reply:
The current style is a mix of cases. Probably fine as is unless it goes to procedural section of choices.
![image](https://user-images.githubusercontent.com/5432499/191623948-2cb3f492-daf5-4d8a-8e70-c8d3b71e7207.png)

Reply:
Added sig-ui-ux as well